### PR TITLE
Package refactor

### PR DESCRIPTION
**Pull request by Mark-Labuschagne**
Note that I have had to refactor the github actions for running the tests, so pay close attention to those files to ensure that you're happy the tests are running correctly.

### CODE REVIEW

**Review by CastelloG — APPROVED** [review]

**Review by MattyJ007 — COMMENTED** [review]
LGTM 👍🏼

File: ohiecli/cmd/docker/docker.go [hunk 1]
```diff
@@ -520,3 +506,22 @@ var untarPackage = func(tarContent io.ReadCloser) (pathToPackage string, err err
 
 	return pathToPackage, nil
 }
+
+// TODO: Check references
```
Comment (michaelloosen) [inline ohiecli/cmd/docker/docker.go:510]: Does this comment still need to be here?

File: .github/workflows/test-go-cli.yaml [hunk 2]
```diff
@@ -5,32 +5,32 @@ on:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (michaelloosen) [inline .github/workflows/test-go-cli.yaml:8]: Do we need to pin this to this version? Can't we rather use the newer version of Ubuntu seeing one of our projects is already on this new version?

**Review by michaelloosen — COMMENTED** [review]
Looks good, minor feedback.

File: ohiecli/cmd/docker/docker.go (hunk 1 above)
Comment (Mark-Labuschagne) [reply ohiecli/cmd/docker/docker.go]: Remove in [70b02e8](https://github.com/openhie/package-starter-kit/pull/14/commits/70b02e8ab62dae04a74398dff39d2402b597a37f) :+1:

File: .github/workflows/test-go-cli.yaml (hunk 2 above)
Comment (Mark-Labuschagne) [reply .github/workflows/test-go-cli.yaml]: Updated to 22.04 in [70b02e8](https://github.com/openhie/package-starter-kit/pull/14/commits/70b02e8ab62dae04a74398dff39d2402b597a37f)

**Review by michaelloosen — APPROVED** [review]
LGTM

**Pull request merged**